java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -0,0 +1,2943 @@
+/*
+ *
+ *  Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package ai.rapids.cudf;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static ai.rapids.cudf.HostColumnVector.OFFSET_SIZE;
+
+/** This class represents the column_view of a column analogous to its cudf cpp counterpart.
```

Comment:
nit: This just feels off to me.  We start the test right after the `/**` which I don't see all that often except when we want a single line java doc, and then we have an extra blank line at the end of the comment.